src/Template/Layout/js/app/mixins/paginated-content.js

```diff
@@ -14,6 +16,9 @@ export const DEFAULT_PAGINATION = {
 export const PaginatedContentMixin = {
     data() {
         return {
+            requestsQueue: [], // array of queued fetch requests
+            requestController: new AbortController(), // AbortController instance
```

Comment:
where is `AbortController` defined? not found in src... Maybe I lost it during merge from master?